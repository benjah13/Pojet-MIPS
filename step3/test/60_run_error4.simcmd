#Commande a lancer dans le script 60_run_error4

run 0xa 1
