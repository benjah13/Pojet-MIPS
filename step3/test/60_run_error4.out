Run only takes one argument!
